NOTICE.md

```diff
@@ -0,0 +1,47 @@
+# Notices for Eclipse Tractus-X
```

Comment:
That's not the current state, see https://github.com/eclipse-tractusx/portal/blob/main/NOTICE.md, for instance the section for Declared Project Licenses is not up to date.

Reply:
Addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/33b9f3814158b68e664ee0efae999956111fe5b2